test/e2e_node/inode_eviction_test.go

```diff
@@ -206,6 +206,12 @@ func runEvictionTest(f *framework.Framework, testCondition string, podTestSpecs
 		})
 
 		AfterEach(func() {
+			By("deleting pods")
```

Comment:
We should not delete pods here directly.

We should:
1) check whether disk pressure will be relieved.
2) check whether we can create a new pod.
3) delete all pods in `AfterEach` to make sure everything will be cleaned up no matter the test success or not.

You can reference how disk eviction test is implemented https://github.com/dashpole/kubernetes/blob/31653aff504724438831290677e5bb69effd5b77/test/e2e_node/disk_eviction_test.go.